### ISSUE TITLE
EnOI evaluation of static perturbations using the LETKF

### ISSUE THREAD

**Issue by guillaumevernieres** [NOAA-EMC/godas#430]
Similar to #429 but using the LETKF.
We should expect a better understanding of:
- The quality of the static ensemble to represent background errors
- Profiling of the LETKF without the cost of running an ensemble

**Comment by liuxiao37k** [comment]
I plan to first discuss with @kbhargava to get the latest updates on soca team's LETKF development.

**Comment by guillaumevernieres** [comment]
> I plan to first discuss with @kbhargava to get the latest updates on soca team's LETKF development.

Yes, no worries/rush, just something that we should start thinking about.

**Comment by liuxiao37k** [comment]
What I remember from last Thursday's discussion:
Technical steps of running a LETKF cycle using the static perturbations that are currently used in EnVAR (better just work off /SCRATCH):
1. recenter the corresponding ensemble forecasts on a single analysis restart from EnVAR;
2. checkpoint application (or should this be the 1st step?);
3. LETKF observer;
4. LETKF solver;
5. compare increment fields from the same cycle between LETKF vs EnVAR, and, if makes some sense:
6. forecasts and postprocessing;
7. repeat the steps for multiple cycles.
@guillaumevernieres anything to add or correct? thanks!

**Comment by guillaumevernieres** [comment]
@liuxiao37k , for point 2, if by checkpoint you mean insert the new state in the restarts, you don't need to.
